### PR TITLE
Raise unit tests for src/proxy/http/remap from the dead.

### PR DESCRIPTION
**Pull request by ywkaras**

### CODE REVIEW

**Comment by ywkaras** [comment]
The plugin load unit tests are not currently working.

File: include/proxy/http/remap/PluginFactory.h [hunk 1]
```diff
@@ -31,8 +31,6 @@
 #include "proxy/http/remap/PluginDso.h"
 #include "proxy/http/remap/RemapPluginInfo.h"
 
-#include "tscore/Ptr.h"
```
Comment (ywkaras) [inline include/proxy/http/remap/PluginFactory.h:34]: Unrelated cleanup.

**Review by cmcfarlen — APPROVED** [review]
Thanks for adding these!

**Comment by cmcfarlen** [comment]
Cherry-picked to v10.0.x
